index.bs

```diff
@@ -555,7 +555,7 @@ General Subsample Encryption constraints {#subsample-encryption}
 
 Within protected samples, the following constraints apply:
 - <assert>Protected samples SHALL be exactly spanned by one or more contiguous subsamples</assert>.
-	- <assert>An OBU MAY be spanned by one or more subsamples</assert>, especially when it has multiple ranges of protected data. However, as recommended in [[!CENC]], writers SHOULD reduce the number of subsamples as possible. This can be achieved by using a subsample that spans multiple consecutive unprotected OBUs as well as the first unprotected and protected parts of the following protected OBU, if such protected OBU exists.
+	- <assert>An OBU MAY be spanned by one or more subsamples</assert>, especially when it has multiple ranges of protected data. However, as recommended in [[!CENC]], subsample encryption entries SHOULD be as compactly represented as possible. For example, only one subsample entry should be created for multiple consecutive unprotected OBUs as well as the first unprotected and protected parts of the following protected OBU, if such protected OBU exists.
```

Comment:
Should we delete the word "entry" in "only one subsample entry"? Or perhaps say "only one subsample encryption entry"?

I am not familiar with these terms, but I noticed that the term "subsample entry" has not been used in this document before, but other similar-looking terms are used, such as "sample entry" and "subsample". The term "subsample encryption entry" is used in `[[!CENC]]` (and copied in this pull request).